File: controllers/gps_controller.py
from flask import jsonify, request

def update_location():
    data = request.get_json()
    latitude = data.get('latitude')
    longitude = data.get('longitude')
    if latitude and longitude:
        return jsonify({'success': True, 'message': 'Localização atualizada com sucesso'}), 200
    return jsonify({'success': False, 'message': 'Dados de GPS inválidos'}), 400

File: backend/src/controllers/GpsController.js
// /backend/src/controllers/GpsController.js
exports.updateLocation = (req, res) => {
    const { latitude, longitude } = req.body;
    // Lógica para processar os dados do GPS e atualizar a localização do veículo no mapa
    res.json({ success: true, message: 'Localização atualizada com sucesso' });
  };
  